app/src/accounts/NextSteps/InlineCollectorCredentials.tsx

```diff
@@ -70,7 +70,14 @@ export default function InlineCollectorCredentials() {
           setAnyCollectorCredentials(collectorCredentials.length > 0);
           setInFlight(false);
         });
-    }, [apiClient, setInFlight, setAnyCollectorCredentials]),
+    }, [
+      apiClient,
+      inFlight,
+      setInFlight,
+      anyCollectorCredentials,
+      setAnyCollectorCredentials,
+      accountId,
+    ]),
```

Comment:
Looks like I could go further and drop the setters from the dependency list, but I'll leave it alone.

> The set functions returned by useState also have stable identity, so you will often see them omitted from the dependencies too. If the linter lets you omit a dependency without errors, it is safe to do.

https://react.dev/learn/synchronizing-with-effects#why-was-the-ref-omitted-from-the-dependency-array